### DISCUSSION TITLE
Camera E27 chip AK3918EN088 V200

### DISCUSSION THREAD

**Discussion by compdima** [e27-camera-hack/E27-Camera-Hack#5] [Q&A]
Добрый день.
Вопрос.
Купил камеру-лампочка с чипом AK3918EN088 V200 версия ПО HW_AWT3710D_XHR_V1.0_WF20231113, EV2W_VA3_V1.3.6.1_20231113.
Так вот - все телодвижение с записями разных патчей на SD-карту не помогали ни чем.
Оставался открытым только порт 8800 и соответственно работала только с прогой V380.
Может кто уже как то разобрались с данным чипом?
Возможно ли - например перепрошить?
------------------------------
Good afternoon.
Question.
I bought a light bulb camera with AK3918EN088 V200 chip, software version HW_AWT3710D_XHR_V1.0_WF20231113, EV2W_VA3_V1.3.6.1_20231113.
So - all the physical movements with recordings of different patches on the SD card did not help in any way.
Only port 8800 remained open and, accordingly, only worked with the V380 program.
Maybe someone has already sorted out this chip?
Is it possible - for example, to reflash?

**Comment by filder35** [comment]
Добрый день. А что вы хотите от этой камеры. Мы вот здесь занимались такими камерами.

https://github.com/e27-camera-hack/E27-Camera-Hack/discussions
А по версии 200 здесь https://github.com/e27-camera-hack/E27-Camera-Hack/discussions/1
